examples/shared/build/build-entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -e
 if [[ $(id -u envoybuild) != "${BUILD_UID}" ]]; then
     usermod -u "${BUILD_UID}" envoybuild
     chown envoybuild /home/envoybuild
+    chown envoybuild /output
```

Comment:
i think this needs to happen irrespective of the usermod - ie outside the `if`

Reply:
Nice. Done